pkg/agent/multicluster/mc_route_controller.go

```diff
@@ -95,8 +122,16 @@ func NewMCDefaultRouteController(
 		ciImportListerSynced:         ciImportInformer.Informer().HasSynced,
 		queue:                        workqueue.NewNamedRateLimitingQueue(workqueue.NewItemExponentialFailureRateLimiter(minRetryDelay, maxRetryDelay), "gatewayroute"),
 		installedCIImports:           make(map[string]*mcv1alpha1.ClusterInfoImport),
-		enableStretchedNetworkPolicy: enableStretchedNetworkPolicy,
-		enablePodToPodConnectivity:   enablePodToPodConnectivity,
+		namespace:                    multiclusterConfig.Namespace,
+		enableStretchedNetworkPolicy: multiclusterConfig.EnableStretchedNetworkPolicy,
+		enablePodToPodConnectivity:   multiclusterConfig.EnablePodToPodConnectivity,
+		serviceCIDRProvider:          serviceCIDRProvider,
+	}
+	if multiclusterConfig.TrafficEncryptionMode == TrafficEncryptionModeWireGuard {
+		controller.wireGuardConfig = &config.WireGuardConfig{
+			Port: multiclusterConfig.WireGuard.Port,
```

Comment:
we have removed the port parameter, so this if condition has removed.

Reply:
I'm not sure if we can do initialization in cmd/antrea-agent/options.go, since we just do validation in that file. Maybe we could add a complete function?

Reply:
I think it can be done in options.go, you can check other feature's validation in the file.

Reply:
Added a new function `setMulticlusterDefault` in `options.go`.